tools/create_packages.sh

```diff
@@ -2,32 +2,35 @@
 
 set -e
 
+echo "Packaging"
+
 # This creates a version such as "v1.2.3-5-g123abc".
-version=`git describe --always --tags`
+VERSION=`git describe --always --tags`
```

Comment:
So capitalizing variables in bash files is just taste, right?

Reply:
Yes, I thought it was the norm. And I realize that the google conventions go for `lower_snake_case` -_-. Which surprises me because I'm quite sure I see `UPPERCASE` 95% of the time everywhere.

Reply:
I think I followed this: https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization

Reply:
I won't change all files now but I'll do that gradually in the future.